### PR TITLE
Release adjustments for v0.6.2

### PR DESCRIPTION
**Pull request by romanlutz**
This PR updates the versions for the release branch. There will be a separate PR for the `main` branch.

### CODE REVIEW

**Review by riedgar-ms — APPROVED** [review]

File: docs/static_landing_page/index.html [hunk 1]
```diff
@@ -27,30 +27,30 @@
...
 		          </li>
 				  <li class="nav-item px-1">
 					<!-- Versioned Link -->
-		            <a class="nav-link" href="./v0.6.1/faq.html">FAQ</a>
+		            <a class="nav-link" href="./v0.6.2/faq.html">FAQ</a>
 		          </li>
 				  <li class="nav-item px-1">
 					<!-- Versioned Link -->
-		            <a class="nav-link" href="./v0.6.1/about/index.html">About Us</a>
+		            <a class="nav-link" href="./v0.6.2/about/index.html">About Us</a>
```
Comment (adrinjalali) [inline docs/static_landing_page/index.html:53]: are these duplicated in two PRs? Ideally they shouldn't

**Review by adrinjalali — APPROVED** [review]

File: docs/static_landing_page/index.html (hunk 1 above)
Comment (romanlutz) [reply docs/static_landing_page/index.html:53]: Yes I followed the release instructions. There is some variation but also a little overlap

Comment (riedgar-ms) [reply docs/static_landing_page/index.html:53]: With the current setup, you don't actually need to update the docs on the branch, only `__init__.py` and the `ReadMe.md`. The docs are only built from `main`.

Comment (romanlutz) [reply docs/static_landing_page/index.html:53]: Right, I got unnecessarily nervous about that. :-)

**Pull request merged**